lib/data_update_scripts/20220105112823_migrate_logo_svg_data.rb

```diff
@@ -0,0 +1,30 @@
+module DataUpdateScripts
+  class MigrateLogoSvgData
+    def run
+      return if ::Settings::General.try(:logo_svg).blank? ||
+        (::Settings::General.try(:original_logo).present? && ::Settings::General.try(:resized_logo).present?)
+
+      Tempfile.create(["logo", ".svg"]) do |file|
+        file.write(::Settings::General.logo_svg)
+
+        logo_svg_uploader = LogoSvgUploader.new.tap do |uploader|
+          uploader.store!(file)
+        end
+
+        ::Settings::General.original_logo = logo_svg_uploader.url
+        ::Settings::General.resized_logo = logo_svg_uploader.resized_logo.url
+      end
+    rescue StandardError => e
+      # If we can't convert the logo, then alert ourselves so that we can track the Forems
+      # whose logos could not be converted.
+
+      Rails.logger.error("Could not convert logo_svg for #{Settings::Community.community_name} Forem to a PNG.")
+
+      Honeybadger.notify(e, context: {
```

Comment:
I would love a second pair of eyes on this from someone that is familiar with Honeybadger. 

We are expecting some logo conversions to fail and we want to be able to easily identify the failures in Honeybadger and notify the Forems that they need to do the conversion manually. Hence I've added the context and then a tag to easily filter these errors.

Reply:
@jgaskins might be a good reviewer of this

Reply:
Thanks Jeremy, I think Jamie is on PTO but I've requested his review as well.